compose/foundation/foundation/api/desktop/foundation.api

```diff
@@ -1904,6 +1904,11 @@ public final class androidx/compose/foundation/text/contextmenu/builder/TextCont
 	public final fun separator ()V
 }
 
+public final class androidx/compose/foundation/text/contextmenu/builder/TextContextMenuBuilderScope_desktopKt {
+	public static final fun item (Landroidx/compose/foundation/text/contextmenu/builder/TextContextMenuBuilderScope;Ljava/lang/Object;Ljava/lang/String;ZLkotlin/jvm/functions/Function3;Lkotlin/jvm/functions/Function1;)V
```

Comment:
Are we sure that it's a final, desktop-only shape?

> We can't put this in common because it's not supported on android or ios.

What exactly is not supported? Does it really depend on a "compilation target" but not on capabilities of underlaying platform ui toolkit or device type?

Reply:
Well. I don't think that it's a question. I'm sure that we should NOT introduce any public API for specific compilation targets at `foundation` level

Reply:
> It's not always the case

What happens when a developer uses this API and then runs the app on a platform that doesn't support it?

> Android is always a special case in AOSP

We can't live in a world where we are not allowed to do the things Android is allowed.

`TextContextMenuBuilderScope.addComponent` is `internal`. So we can't put this extension function in `ui`.